hack/test-cmd.sh

```diff
@@ -343,6 +343,16 @@ echo "imageRepositoryMappings: ok"
 # the local image repository takes precedence over the Docker Hub "mysql" image
 [ "$(osc new-app mysql -o yaml | grep mysql-55-centos7)" ]
 osc new-app php mysql
+# check if we can create from a stored template
+osc create -f examples/sample-app/application-template-stibuild.json
+osc get template ruby-helloworld-sample
+[ "$(osc new-app ruby-helloworld-sample -o yaml | grep MYSQL_USER)" ]
+[ "$(osc new-app ruby-helloworld-sample -o yaml | grep MYSQL_PASSWORD)" ]
+[ "$(osc new-app ruby-helloworld-sample -o yaml | grep ADMIN_USERNAME)" ]
+[ "$(osc new-app ruby-helloworld-sample -o yaml | grep ADMIN_PASSWORD)" ]
+# create from template with code explicitly set is not supported
+[ ! "$(osc new-app ruby-helloworld-sample~git@github.com/mfojtik/sinatra-app-example)" ]
```

Comment:
Actually we use it in osc help.  We need a good ootb example we can use in the docs - what's a good one?

> On Apr 17, 2015, at 10:52 AM, Michal Fojtik <notifications@github.com> wrote:
> 
> In hack/test-cmd.sh:
> 
> > @@ -343,6 +343,16 @@ echo "imageRepositoryMappings: ok"
> >  # the local image repository takes precedence over the Docker Hub "mysql" image
> >  [ "$(osc new-app mysql -o yaml | grep mysql-55-centos7)" ]
> >  osc new-app php mysql
> > +# check if we can create from a stored template
> > +osc create -f examples/sample-app/application-template-stibuild.json
> > +osc get template ruby-helloworld-sample
> > +[ "$(osc new-app ruby-helloworld-sample -o yaml | grep MYSQL_USER)" ]
> > +[ "$(osc new-app ruby-helloworld-sample -o yaml | grep MYSQL_PASSWORD)" ]
> > +[ "$(osc new-app ruby-helloworld-sample -o yaml | grep ADMIN_USERNAME)" ]
> > +[ "$(osc new-app ruby-helloworld-sample -o yaml | grep ADMIN_PASSWORD)" ]
> > +# create from template with code explicitly set is not supported
> > +[ ! "$(osc new-app ruby-helloworld-sample~git@github.com/mfojtik/sinatra-app-example)" ]
> 
> 
> —
> Reply to this email directly or view it on GitHub.
>